src/components/EventForm.tsx

```diff
@@ -239,12 +239,20 @@ const EventForm = ({
           <option disabled value="">
             Timezone
           </option>
-          <option>PST (Pacific Standard Time)</option>
-          <option>EST (Eastern Standard Time)</option>
-          <option>GMT (Greenwich Mean Time)</option>
-          <option>CET (Central European Time)</option>
-          <option>IST (Indian Standard Time)</option>
-          <option>JST (Japan Standard Time)</option>
+          <option value="PST">PST (Pacific Standard Time)</option>
```

Comment:
In the text, can you add the UTC offset. Like UTC-8 for PST and then order by that number. Default to PST maybe?

Reply:
The current ordering is hard to find a specific value since it isn't alphabetic or time ordered.

Reply:
UTC offsets added to the time zones. Added function to sort by UTC times from negative to positive (Ex. UTC-08:00 will be higher on the list than UTC+08:00).